src/marketplace/Marketplace.js

```diff
@@ -57,18 +57,10 @@ class Marketplace extends React.Component<Props, State> {
 			case 'NONE':
 				partView=(<h2>Select a type on the left</h2>);
 				break;
-			}
+		}
 		return (
 	  		<div id="Parent">
-				<div className="row">
-		  			<div className="col-md-4">
-						<Link to="/MainMenu">
-		  					<button type="button" className="btn btn-secondary btn-lg">&lt;- Back to Main Menu</button>
-						</Link>
-	  				</div>
-		  			<h1 className="col-md-4 text-center">Marketplace</h1>
-		  			<h4 className="col-md-4 text-right">Currency: $9432</h4>
-				</div>
+				<Navbar styleName="navbtn" linkName="Mainmenu" returnName="Back to Main Menu" pageName="Marketplace" userName="FRIcker | $465128"/>
 				<div className="row mt-5">
 		  			<div className="col-md-4">
 					  <div className="list-group col-md-6">
```

Comment:
Indent line 87 (old) / 79 (new) containing `{partView}` since it is inside a div tag please